src/components/tasks/TaskSearch/TaskSearch.tsx

```diff
@@ -1,8 +1,15 @@
-import { useState } from 'react';
-import { LuChevronDown } from 'react-icons/lu';
+import { useEffect, useState, useRef } from 'react';
 import styles from './tasksearch.module.scss';
 import { TABS, Tab } from '@/interfaces/task.type';
-import FilterModal from './FilterModal';
+import FilterDropdown from './FilterDropdown';
+import useDebounce from '@/hooks/useDebounce';
+import { TaskSearchOption } from '@/interfaces/searchOptions.type';
+import Options from './Suggestion/Options';
+import RenderPills from './Suggestion/Pill';
```

Comment:
Any reason for naming this RenderPills and not just Pill ?

Reply:
Done